tests/blockchain/eth/contracts/integration/test_integration.py

```diff
@@ -449,7 +470,7 @@ def test_all(testerchain, token, escrow, policy_manager, government):
     testerchain.wait_for_receipt(tx)
     tx = government.functions.vote(False).transact({'from': ursula2})
     testerchain.wait_for_receipt(tx)
-    tx = user_escrow_1.functions.vote(True).transact({'from': ursula3})
+    tx = user_escrow_proxy_1.functions.vote(True).transact({'from': ursula3})
     testerchain.wait_for_receipt(tx)
 
     # Can't vote again
```

Comment:
@szotov Here is a good example.
Explain why we can't vote again instead of simply saying, "Can't vote again".

A good rule of thumb I like to use is if I am explaining _what_ something is doing, I'm doing it wrong. I should be explaining _why_ I'm doing something. Sometimes both are needed, but most of the time, I want to read _why_ something is happening.